pkg/registry/core/serviceaccount/storage/token_test.go

```diff
@@ -42,46 +42,46 @@ import (
 func TestCreate_Token_WithExpiryCap(t *testing.T) {
 
 	testcases := []struct {
-		desc                 string
-		extendExpiration     bool
-		maxExpirationSeconds int
-		expectedTokenAgeSec  int
-		isExternal           bool
+		desc                         string
+		extendExpiration             bool
+		maxExpirationSeconds         int
+		expectedTokenAgeSec          int
+		maxExtendedExpirationSeconds int
 	}{
 		{
-			desc:                 "maxExpirationSeconds honoured",
-			extendExpiration:     true,
-			maxExpirationSeconds: 5 * 60 * 60, // 5h
-			expectedTokenAgeSec:  5 * 60 * 60, // 5h
-			isExternal:           true,
+			desc:                         "maxExpirationSeconds honoured",
+			extendExpiration:             false,
```

Comment:
don't change the fundamentals of this test case... this scenario was ensuring that extended tokens are issued _shorter_ than ExpirationExtensionSeconds when the external signer has a shorter max

To replicate that intent, this test case should be:
```diff
		{
+			desc:                         "maxExtendedExpirationSeconds honoured",
-			desc:                         "maxExpirationSeconds honoured",
			extendExpiration:     true,
+			maxExpirationSeconds:         2 * 60 * 60,                               // 2h
-			maxExpirationSeconds:         5 * 60 * 60,                               // 5h
			expectedTokenAgeSec:  5 * 60 * 60, // 5h
-			isExternal:           true,
+			maxExtendedExpirationSeconds: 5 * 60 * 60,                               // 5h
		},
```

Reply:
Done